records_mover/db/vertica/unloader.py

```diff
@@ -119,15 +119,14 @@ def unload_to_s3_directory(self,
         return export_count
 
     def known_supported_records_formats_for_unload(self) -> List[BaseRecordsFormat]:
-        if self.s3_available():
-            return [DelimitedRecordsFormat(variant='vertica')]
-        else:
-            return []
+        return [DelimitedRecordsFormat(variant='vertica')]
```

Comment:
This was an existing kludge to try to accomplish some of the "negotiation" happening with this PR.  Now that we have it more fully baked at the move() algorithm level, we can tear down this hack, as it's no longer needed.